packages/teams-js/src/public/search.ts

```diff
@@ -127,4 +145,74 @@ export namespace search {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.search ? true : false;
   }
+
+  /**
+   * Clear the host M365 application's search box
+   *
+   * @beta
+   */
+  export function closeSearch(): Promise<void> {
+    return new Promise<void>((resolve) => {
+      ensureInitialized(runtime, FrameContexts.content);
+      if (!isSupported()) {
+        throw new Error('Not supported');
+      }
+
+      resolve(sendAndHandleError('search.closeSearch', version));
+    });
+  }
+
+  /**
+   * @hidden
+   * Undocumented function used to set a mock window for unit tests
+   *
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  export function _initialize(hostWindow: any): void {
+    Communication.currentWindow = hostWindow;
+  }
+
+  /**
+   * @hidden
+   * Undocumented function used to clear state between unit tests
+   *
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  export function _uninitialize(): void {
```

Comment:
Same with this: unit test only functions should live in unit test files and not in the public exported capabilities

Reply:
This one also